readme/tutorial/FAQ.scalatex

```diff
@@ -0,0 +1,49 @@
+@import scalaworld.Tutorial._
+@import scalaworld.Readme._
+
+@sect{FAQ}
+  @sect{Macros?}
+    Originally, Scalameta was founded to become a better macro system for Scala,
+    but over time we shifted focus to developer tools and spun off the new macro system
+    into a separate project: @lnk("scalamacros/scalamacros", "https://github.com/scalamacros/scalamacros").
+    See @paradise for more information about the deprecated scalameta macro annotations.
+
+  @sect{What is the quasiquote for X?}
+    Here is an overview of quasiquote syntax: @lnk("https://github.com/scalameta/scalameta/blob/master/notes/quasiquotes.md").
+
+  @sect{Can I use Scalameta with Scala.js?}
+    Yes, the main Scalameta modules support Scala.js.
+
+
+  @sect{I'd like to use the semantic api as a replacement for scalac's
+        presentation compiler, is this doable? intended usage?}
+    It's not doable and it's not the intended usage. The prime application we
```

Comment:
"In principle, this should be doable, but it requires custom development and is not supported at the moment."?